containers/ecr-viewer/tests/integration/programAreaService.test.ts

```diff
@@ -105,17 +120,30 @@ describe("program area service", () => {
 
     // program with name already exists
     jest.spyOn(console, "error").mockImplementation();
-    await expect(createProgramArea(progName, [])).rejects.toThrow();
+    await expect(
+      createProgramArea({ name: progName, conditions: [] }),
+    ).rejects.toThrow();
   });
 
   it("should update a program area name", async () => {
     const progName = "Sad Times";
-    const id = await createProgramArea(progName, ["123"]);
+    const id = await createProgramArea({ name: progName, conditions: ["123"] });
```

Comment:
```suggestion
    const uuiid = await createProgramArea({ name: progName, conditions: ["123"] });
```

super minor takeitorleaveit nit - if you name this uuid, then you can just do `uuid` instead of `uuid: id` in the places